src/dotnet/HQ.Server/Controllers/TimeEntriesControllerV1.cs

```diff
@@ -0,0 +1,107 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using Asp.Versioning;
+using FluentResults.Extensions.AspNetCore;
+using HQ.Abstractions.Common;
+using HQ.Abstractions.Times;
+using HQ.API;
+using HQ.Server.Authorization;
+using HQ.Server.Data;
+using HQ.Server.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+
+namespace HQ.Server.Controllers
+{
+    [Authorize]
+    [ApiController]
+    [ApiVersion(1.0)]
+    [Tags("TimeEntries")]
+    [Route("v{version:apiVersion}/TimeEntries")]
+    [ProducesResponseType<List<ErrorSummaryV1>>(StatusCodes.Status400BadRequest)]
+    public class TimeEntriesControllerV1 : ControllerBase
```

Comment:
Instead of using the StaffId to perform authorization checks, we should utilize the `_authorizationService` to check our `TimeEntryOperation` actions against the relevant time entry record.

A new authorization handler will need to be implemented for time entry checks (Check the Authorization folder for how we do PSR checks).